src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {` ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          I am
+          {` ${age}`}
+        </p>
+      )}
+
+      {
+        (isMarried && (
+          <p className="Person__partner">
+            {
+              sex === 'm'
+                ? `${partnerName} is my wife`
+                : `${partnerName} is my husband`
```

Comment:
you need to render wife/husband by condition here, do not duplicate code
but when you will render wife/husband by condition, do not use nested ternary, move it to some variable